Library/Homebrew/shims/super/cc

```diff
@@ -72,7 +72,11 @@ class Cmd
     else
       # Note that this is a universal fallback, so that we'll always invoke
       # HOMEBREW_CC regardless of what name under which the tool was invoked.
-      ENV["HOMEBREW_CC"]
+      if ENV["HOMEBREW_CC"] == "llvm_clang"
+        "#{ENV["HOMEBREW_PREFIX"]}/opt/llvm/bin/clang"
```

Comment:
Yep. This patch addresses the case when `HOMEBREW_CC == llvm_clang` and `super/cc` is invoked as any compiler other than `llvm_clang`, for example `clang` or `gcc`. Line 61 tests whether `arg0 == llvm_clang` but not `HOMEBREW_CC == llvm_clang`.